java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsContentAccess.ql

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
I think this should be integrated in the other query as a disjunct, instead of being a standalone query.